semantickernel-experimental/src/main/java/com/microsoft/semantickernel/connectors/data/azureaisearch/AzureAISearchVectorStoreRecordCollection.java

```diff
@@ -194,7 +193,7 @@ public Mono<Record> getAsync(
         @Nonnull String key, GetRecordOptions options) {
         // If vectors are not requested, only fetch non-vector fields
         List<String> selectedFields = null;
-        if (options != null && !options.includeVectors()) {
+        if (options == null || !options.isIncludeVectors()) {
```

Comment:
this will effectively default to not including vectors, is that correct?

Reply:
Yes, that's right